src/ovl2/ovl2_3.c

```diff
@@ -2713,48 +2715,48 @@ void *func_800FCE50_ovl2(void) {
 }
 
 void func_800FCED8_ovl2(UNUSED s32 arg0) {
-    func_800B1900_ovl2(D_8004A7C4->str.unk2);
+    func_800B1900_ovl2(D_8004A7C4->unk0.asU16[1]);
```

Comment:
I hit this problem before; it turns out the hiLo union isn't even necessary, and all you have to do is cast unk0 to u16 and the compiler will automatically access the 0x2 offset instead. I keep the union in the file just in case something actually explicitly requires the 0x2 offset